### PR TITLE
fix: when building with enable_plugins=false

### PR DESCRIPTION
**Pull request by miniak**
#### Description of Change
Fix building with `enable_plugins=false`

#### Checklist
- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

#### Release Notes
Notes: no-notes

### CODE REVIEW

File: patches/chromium/fix_when_building_with_enable_basic_printing_false.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Milan Burda <milan.burda@gmail.com>
+Date: Sat, 28 Sep 2019 10:39:58 +0200
+Subject: fix: when building with enable_basic_printing=false
+
+
```
Comment (nornagon) [inline patches/chromium/fix_when_building_with_enable_basic_printing_false.patch]: Patches must include justification. Also, it seems like this patch should join the other printing mega-patch?

File: patches/chromium/fix_when_building_with_enable_plugins_false.patch [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ namespace content {
+ 
+ namespace {
+@@ -152,7 +156,9 @@ void SetupPPAPISandboxParameters(sandbox::SeatbeltExecClient* client) {
+   SetupCommonSandboxParameters(client);
+ 
+   std::vector<content::WebPluginInfo> plugins;
++#if BUILDFLAG(ENABLE_PLUGINS)
+   PluginService::GetInstance()->GetInternalPlugins(&plugins);
++#endif
```
Comment (nornagon) [inline patches/chromium/fix_when_building_with_enable_plugins_false.patch]: Should this be upstreamed?

File: patches/chromium/fix_when_building_with_enable_basic_printing_false.patch (hunk 1 above)
Comment (miniak) [reply patches/chromium/fix_when_building_with_enable_basic_printing_false.patch]: merged with `printing.patch`

File: patches/chromium/fix_when_building_with_enable_plugins_false.patch (hunk 2 above)
Comment (miniak) [reply patches/chromium/fix_when_building_with_enable_plugins_false.patch]: @deepak1556 can you pleas help with this?

**Comment by miniak** [comment]
@nornagon do you want us to upstream the Chromium patch before merging this PR?

**Comment by deepak1556** [comment]
Patch uploaded at https://chromium-review.googlesource.com/c/chromium/src/+/1845624

**Comment by deepak1556** [comment]
Have updated the patches from upstream. Should be good to go once build is finished.

**Pull request merged**